fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -369,10 +375,16 @@ def build_settings(key: nil, optional: true)
 
         # SwiftPM support
         if FastlaneCore::Helper.xcode_at_least?('11.0')
-          UI.important("Resolving Swift Package Manager dependencies...")
-          FastlaneCore::CommandExecutor.execute(command: build_xcodebuild_resolvepackagedependencies_command,
-                                                print_all: true,
-                                                print_command: !self.xcodebuild_list_silent)
+          if command = build_xcodebuild_resolvepackagedependencies_command
```

Comment:
```suggestion
          if (command = build_xcodebuild_resolvepackagedependencies_command)
```

This fixes rubocop 😊